### ISSUE TITLE
Failed to parse --problems option at v0.1.7

### ISSUE THREAD

**Issue by c-bata** [sile/kurobako#5]
I bumped into the following error at https://github.com/c-bata/goptuna/pull/80.

```
$ kurobako studies --solvers $(kurobako solver random) --problems $(kurobako problem sigopt --dim 5 ackley) --repeats 100 --budget 100
error: Invalid value for '--problems <problems>...': InvalidInput (cause; missing field `int` at line 1 column 36)
HISTORY:
  [0] at kurobako_core/src/json.rs:14

$ kurobako studies --solvers '{"random":{}}' --problems '{"sigopt":{"name":"ACKLEY","dim":5}}' --repeats 100 --budget 100          
error: Invalid value for '--problems <problems>...': InvalidInput (cause; missing field `int` at line 1 column 36)
HISTORY:
  [0] at kurobako_core/src/json.rs:14

```

I confirmed it works on kurobako v0.1.5.

```
(venv) $ kurobako-0.1.5 studies --solvers '{"random":{}}' --problems '{"sigopt":{"name":"ACKLEY","dim":5}}' --repeats 10 --budget 100 
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}

```

**Comment by c-bata** [comment]
It seems this field is required.

https://github.com/sile/kurobako/blob/775accba1b49ce75173de99ff10bcf308b2b870e/kurobako_problems/src/sigopt.rs#L42-L45

I don't know what should I pass. But it looks the following command works as expected.

```
$ kurobako studies --solvers '{"random":{}}' --problems '{"sigopt":{"name":"ACKLEY","dim":5, "int": [1]}}' --repeats 10 --budget 100
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
{"solver":{"random":{}},"problem":{"sigopt":{"name":"ACKLEY","dim":5,"int":[1]}},"budget":100,"concurrency":1,"scheduling":"RANDOM","seed":null}
```

**Comment by sile** [comment]
Thank you for reporting this issue. This is due to my careless mistake... I'll fix this problem soon.

**Comment by sile** [comment]
Fixed and published [v0.1.8](https://crates.io/crates/kurobako/0.1.8).

**Comment by c-bata** [comment]
Thank you. I confirmed it works fine at https://github.com/c-bata/goptuna/pull/80.
